generator-theia/src/extension/extension-package-generator.ts

```diff
@@ -107,4 +108,16 @@ export class ExtensionPackageGenerator extends AbstractGenerator {
             ]
         }
     }
+
+    private getBuildCommand(): string {
+        const concurentValue = "concurrently -n compile,lint -c blue,green \"yarn run compile\" \"yarn run lint\"";
+        let buildCmd = "";
```

Comment:
you could remove that var and just return in the if/else...

Reply:
Done